README.md

```diff
@@ -34,15 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+A: I would tell them that semantic HTML gives your code and the look of your application meaning. It also seperates your code into their own groups and makes thing organized.
```

Comment:
✔️ 
technically semantics is all about the meaning of the markup, usually it is not a guarantee for anything style or look wise although I understand some of the semantic tags may imply certain default looks.